### PR TITLE
Feature/lidar driver

### PR DESCRIPTION
**Pull request by msmcconnell**
# PR Details
## Description
This PR introduces the following changes to the configs
1. The ROS2 lidar driver is used for connecting to the velodyne on all configs
2. A SubsystemControllersParams.yaml file is added to all configs to allow for configuration of fault handling in ROS2
3. For the trucks the bridge is updated to support lidar data concatenation.

## Related Issue
Supports https://github.com/usdot-fhwa-stol/carma-platform/issues/1609
Supports https://github.com/usdot-fhwa-stol/carma-platform/issues/1500

## Motivation and Context
Working ros2 drivers

## How Has This Been Tested?
local testing
TODO testing on the cars

## Types of changes

- [x] Defect fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [x] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
[CARMA Contributing Guide](https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/Contributing.md) 
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: freightliner_cascadia_2012_dot_10002/bridge.yml [hunk 1]
```diff
@@ -9,6 +9,14 @@ topics:
     topic: /hardware_interface/lidar/points_raw
     type: sensor_msgs/msg/PointCloud2
     queue_size: 1
+  -
+    topic: /hardware_interface/velodyne_1/lidar/points_raw
+    type: sensor_msgs/msg/PointCloud2
+    queue_size: 3
+  -
```
Comment (JonSmet) [inline freightliner_cascadia_2012_dot_10002/bridge.yml:16]: Can we add /system_alert -> carma_msgs/msg/SystemAlert to the bridge.yml files as well?

**Review by JonSmet — CHANGES_REQUESTED** [review]

File: freightliner_cascadia_2012_dot_10002/bridge.yml (hunk 1 above)
Comment (msmcconnell) [reply freightliner_cascadia_2012_dot_10002/bridge.yml:16]: Huh, I could have sworn that was already there, but i don't see it. I'll add it

File: chrysler_pacifica_ehybrid_s_2019/SubsystemControllerParams.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+localization:
+  localization_controller:
+    subsystem_namespace: /localization
+    required_subsystem_nodes: []
+    unmanaged_required_nodes: []
+    full_subsystem_required: true
+    sensor_nodes:
+      - /hardware_interface/velodyne_lidar_driver_wrapper_node
+    # TODO update the fault map once ros2 gps is integrated
+    sensor_fault_map: '
```
[the remark this answers is not in the document]
Comment (msmcconnell) [reply chrysler_pacifica_ehybrid_s_2019/SubsystemControllerParams.yaml:44]: What kind of comment of you thinking?

[the remark this answers is not in the document]
Comment (JonSmet) [reply chrysler_pacifica_ehybrid_s_2019/SubsystemControllerParams.yaml:44]: Ah I just re-read the commented lines at the top of this file, which point to the subsystem_controlls/config files in carma-platform for the parameter descriptions. That is fine, and my comment here can be disregarded.

File: freightliner_cascadia_2012_dot_10003/docker-compose.yml [hunk 3]
```diff
@@ -156,6 +156,17 @@ services:
     image: usdotfhwastoldev/carma-velodyne-lidar-driver:develop
     container_name: velodyne-lidar-driver
     network_mode: host
+    volumes_from:
+      - container:carma-config:ro
+    volumes:
+      - /opt/carma/logs:/opt/carma/logs
+      - /opt/carma/.ros:/home/carma/.ros
+    command: bash -c 'source /opt/carma/install/setup.bash && ros2 launch /opt/carma/vehicle/config/drivers.launch.py drivers:=velodyne_lidar_driver_wrapper'
```
Comment (JonSmet) [inline freightliner_cascadia_2012_dot_10003/docker-compose.yml:164]: Should this be `... source /opt/carma/install_ros2/setup.bash ..`?

**Review by JonSmet — CHANGES_REQUESTED** [review]

File: freightliner_cascadia_2012_dot_10003/docker-compose.yml (hunk 3 above)
Comment (msmcconnell) [reply freightliner_cascadia_2012_dot_10003/docker-compose.yml:164]: No this image contains only a ROS2 installation so it doesn't have the separate install vs install_ros2 folders

Comment (JonSmet) [reply freightliner_cascadia_2012_dot_10003/docker-compose.yml:164]: Okay that makes sense. The rest of the PR looks good so I will approve.

**Review by JonSmet — APPROVED** [review]

**Pull request merged**